engines/dfc_provider/app/controllers/dfc_provider/api/v0/base_controller.rb

```diff
@@ -12,6 +12,8 @@ class BaseController < ::Api::V0::BaseController
 
         respond_to :json
 
+        def show; end
```

Comment:
What is this action doing? Shouldn't the BaseController be abstract and nothing should route here?

Reply:
I need to generate the base url for the JSON-LD context. I agree that this is not so nice but I couldn't find a better way. I will harcode it to solve this issue.

Reply:
Would it make sense to create an IndexController, HomeController or ApiController with a show method just for this purpose? Is there anything returned when querying that page? Or maybe RootController? I'm not sure about the purpose in the context.